apps/src/schoolInfo/utils/schoolInfoInvalid.ts

```diff
@@ -0,0 +1,60 @@
+import {
```

Comment:
Is pulling `schoolInfoInvalid` out of `SchoolInfoInterstitial` for decluttering or will it be used elsewhere as well?

Reply:
Okay! I was just curious if all of its associated tests coudl live with the tests for `SchoolInfoInterstitial` or if it would be used separately.

Reply:
good question... I should probably move those tests from `SchoolInfoInterstitialTest` to a separate file just for this util 🤔

Reply:
okay updated!